pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -471,6 +483,18 @@ func FluentBitDefaults(fluentbitSpec *FluentbitSpec) error {
 		if fluentbitSpec.BufferVolumeImage.PullPolicy == "" {
 			fluentbitSpec.BufferVolumeImage.PullPolicy = "IfNotPresent"
 		}
+		if fluentbitSpec.BufferVolumeResources.Limits == nil {
+			fluentbitSpec.BufferVolumeResources.Limits = v1.ResourceList{
+				v1.ResourceMemory: resource.MustParse("10M"),
+				v1.ResourceCPU:    resource.MustParse("1m"),
+			}
+		}
+		if fluentbitSpec.BufferVolumeResources.Requests == nil {
+			fluentbitSpec.BufferVolumeResources.Requests = v1.ResourceList{
+				v1.ResourceMemory: resource.MustParse("5M"),
+				v1.ResourceCPU:    resource.MustParse("1m"),
+			}
+		}
```

Comment:
```suggestion
		if fluentbitSpec.BufferVolumeResources.Limits == nil {
			fluentbitSpec.BufferVolumeResources.Limits = v1.ResourceList{
				v1.ResourceMemory: resource.MustParse("10M"),
				v1.ResourceCPU:    resource.MustParse("50m"),
			}
		}
		if fluentbitSpec.BufferVolumeResources.Requests == nil {
			fluentbitSpec.BufferVolumeResources.Requests = v1.ResourceList{
				v1.ResourceMemory: resource.MustParse("10M"),
				v1.ResourceCPU:    resource.MustParse("1m"),
			}
		}
```

Reply:
done in b68e60adea95bb053946c011f411205cd935b10f